.github/workflows/.hintrc

```diff
@@ -0,0 +1,18 @@
+{
+    "connector": {
+      "name": "local",
+      "options": {
+        "pattern": ["**", "!.git/**", "!node_modules/**"]
+      }
+    },
+    "extends": ["development"],
+    "formatters": ["stylish"],
+    "hints": [
+      "button-type",
+      "disown-opener",
+      "html-checker",
+      "meta-charset-utf-8",
+      "meta-viewport",
+      "no-inline-styles:error"
+    ]
+  }
```

Comment:
- When you copy [.hintrc](https://github.com/microverseinc/linters-config/blob/master/html-css/.hintrc) it must be in the root directory of your project not inside github/workflow folder. You can checkout configuration [here](https://github.com/microverseinc/linters-config/tree/master/html-css). Please fix it.

Reply:
Thanks for your feedback. I have changed the location of the file in the new commit kindly verify if I am on the right track.